tests/journald_tests.cpp

```diff
@@ -351,19 +351,21 @@ TEST_F(JournaldLoggerTest, ROOT_CGROUPS_LaunchThenRecoverThenLaunchNested)
   // Launch the top-level/parent container.
   // We need to checkpoint it so that it will survive recovery
   // (hence the `forked.pid` argument).
-  Future<bool> launch = containerizer->launch(
-      containerId,
-      createContainerConfig(None(), executorInfo, executorRunPath),
-      std::map<std::string, std::string>(),
-      path::join(flags.work_dir,
-          "meta",
-          "slaves", stringify(state.id),
-          "frameworks", stringify(executorInfo.framework_id()),
-          "executors", stringify(executorInfo.executor_id()),
-          "runs", stringify(containerId),
-          "pids", "forked.pid"));
-
-  AWAIT_ASSERT_TRUE(launch);
+  Future<mesos::internal::slave::Containerizer::LaunchResult> launch =
```

Comment:
You could put a `using mesos::internal::slave::Containerizer;` at the top of the test and shorten these namespaces.